src/workloads/docs/Generators.yml

```diff
@@ -125,6 +126,9 @@ Actors:
             # Generate the actor ID as a string
             actorString: {^ActorIdString: {}}
 
+            # Generate a random airport code from a dataset
+            airportDataSet: {^ChooseFromDataset: {"path": "./src/genny/src/workloads/datasets/airports_codes.txt"}}
```

Comment:
Could you include a comment explaining how to properly construct the path?

Reply:
Maybe something short here and reference to the description in RandomStringFromDataset.yml, or just comment "See RandomStringFromDataset.yml for full documentation on this generator".

Reply:
Done, I added the comment to people check the already documented yml.